pkg/chartutils/images.go

```diff
@@ -231,12 +264,12 @@ func pushImage(imgData *imagelock.ChartImage, imagesDir string, o crane.Options)
 	return nil
 }
 
-func getImageTarFile(imagesDir string, dgst imagelock.DigestInfo) string {
-	return filepath.Join(imagesDir, fmt.Sprintf("%s.tar", dgst.Digest.Encoded()))
+func getImageLayourDir(imagesDir string, dgst imagelock.DigestInfo) string {
```

Comment:
You have a typo here in the name of this method.

Reply:
Nice catch.